pkg/controller/cluster/controller.go

```diff
@@ -17,17 +17,30 @@ limitations under the License.
 package cluster
 
 import (
+	"fmt"
+	"time"
+
 	"github.com/kubernetes-incubator/apiserver-builder/pkg/builders"
+	"github.com/kubernetes-incubator/apiserver-builder/pkg/controller"
 
 	"github.com/golang/glog"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
```

Comment:
should these be grouped by stdlib; external packages; internal packages? if yes, add a line here and delete line 25, otherwise it's fine.